code/game/gamemodes/blob/powers.dm

```diff
@@ -237,7 +237,7 @@
 	expand_blob(T)
 
 /mob/camera/blob/proc/expand_blob(turf/T)
-	if(world.time > last_attack)
+	if(world.time < last_attack)
```

Comment:
Remember the smaller part is the eyes and the larger part is its mouth. It must want to get hungry at somepoint.